app/views/content/life-as-a-teacher/explore-subjects/computing/_article.html.erb

```diff
@@ -8,14 +8,12 @@
       <li>digital literacy</li>
       <li>computer science</li>
     </ul>
+    <%= render 'content/shared/teaching/benefits' %>
   </section>
 
   <section class="col col-720">
-    <%= render Content::QuoteComponent.new(
-      text: "Every lesson presents an opportunity to delve into the latest advancements in the field and inspire students to become innovators. Teaching computing is about fostering a deep understanding of technology while enabling students to develop the skills and resilience to thrive in this digital age.",
-      name: "Shiulee, Curriculum lead for computing and business",
-      large: true
-    ) %>
+    <!-- Add chris quote  -->
```

Comment:
I'm not sure if you need these comments really - however, if you prefer to keep them (which is fine!), this one needs updating!

```suggestion
    <!-- Add shiulee quote  -->
```